code/src/WijmoProvider/Features/Export.ts

```diff
@@ -22,6 +23,14 @@ namespace WijmoProvider.Feature {
             );
         }
 
+        private _handleFilename(fileName: string, isCSV = false): string {
+            if (fileName === undefined || fileName === '') {
```

Comment:
Wouldn't be better to set the value for fileName as a default value on the parameter itself?

fileName: string = 'DataGridReactive'

Reply:
That only works for when the filename is _undefined_. If the name is '' (empty), then the default value would not be used...